nixos/modules/services/security/clamav-unofficial-sigs.nix

```diff
@@ -0,0 +1,71 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.clamav-unofficial-sigs;
+in {
+  options.services.clamav-unofficial-sigs = with types; {
+    enable = mkEnableOption "Unofficial ClamAV signatures";
+
+    extraConfig = mkOption {
```

Comment:
By [convention](https://github.com/NixOS/rfcs/pull/42) we should call this option `settings`, much like I did [here](https://github.com/NixOS/nixpkgs/pull/106698).